healthcheck/handler.go

```diff
@@ -16,7 +16,8 @@ func (hc *HealthCheck) Handler(w http.ResponseWriter, req *http.Request) {
 	now := time.Now().UTC()
 	ctx := req.Context()
 
-	hc.Status = hc.getStatus(ctx)
+	newStatus := hc.getAppStatus(ctx)
+	hc.SetStatus(newStatus)
```

Comment:
SetStatus returns a string which I assume is used in the switch statement?..not sure how this works

Reply:
I am ignoring the return value, because I already have it in `newStatus`